packages/wp-now/src/config/providers/CliConfigProvider.ts

```diff
@@ -0,0 +1,34 @@
+import { SupportedPHPVersion } from '@php-wasm/universal';
+import { ConfigProviderInterface } from './ConfigProvider';
+
+export interface CliOptions {
+	php: string;
+	path: string;
+	wp: string;
+}
+
+class CliConfigProvider implements ConfigProviderInterface {
```

Comment:
This class just rewrites one object to another object. `yargs` should be able to output the expected format with a few aliases, removing the need for this class.